scripts/3.2.2_assign_primary_zone_work.py

```diff
@@ -202,7 +200,9 @@ def main(config_file):
     #### ASSIGN TO ZONE FROM FEASIBLE ZONES ####
 
     zone_assignment = WorkZoneAssignment(
-        activities_to_assign=possible_zones_work, actual_flows=travel_demand_dict_nomode
+        activities_to_assign=possible_zones_work,
+        actual_flows=travel_demand_dict_nomode,
+        scaling=config.parameters.part_time_work_prob,
```

Comment:
Currently config of scaling of the flows is implemented through the `part_time_work_prob`. The name of this parameter should be revised - however, this will change the ID of the config so it would be preferred to update the name in a subsequent PR once the current parameter testing is complete.

Reply:
Opened issue #99